services/userService.js

```diff
@@ -0,0 +1,40 @@
+const userDao = require("../models/userDao");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+
+const signUp = async (email, name, password, address, phone, birthdate) => {
+  const saltRounds = 10;
+  const hashPassword = await bcrypt.hash(password, saltRounds);
+  const createUser = await userDao.createUser(
+    email,
+    name,
+    hashPassword,
+    address,
+    phone,
+    birthdate
+  );
+
+  return createUser;
```

Comment:
```suggestion
  return userDao.createUser(
    email,
    name,
    hashPassword,
    address,
    phone,
    birthdate
  );
```
이 부분도 불필요한 변수 선언이므로 바로 return 하도록 해주세요~

Reply:
네 알겠습니다 !

Reply:
누락된 부분도 반영해주세요~!

Reply:
넵 수정했습니다 !